### PR TITLE
Convert README/CHANGES to markdown

### LINKED ISSUE #114: Convert README and CHANGES to markdown

### PR DESCRIPTION
**Pull request by ericof**
Fix #114

### CODE REVIEW

**Comment by stevepiercy** [comment]
I don't know when I can review this. It could take me a couple of weeks, given my current Plone docs backlog. I ask for your patience.

Meanwhile, I would like to know the ultimate goals per https://github.com/plone/plone.app.caching/issues/114#issuecomment-1434671911

Thank you!

**Comment by ericof** [comment]
Moving documentation to [`plone/documentation`](https://github.com/plone/documentation/pull/1449), this PR will become only about README and CHANGES

File: .gitignore [hunk 1]
```diff
@@ -8,11 +8,17 @@ develop-eggs/
...
 lib/
 parts/
 var/
+# Docs
 docs/Makefile
 docs/make.bat
 docs/doctrees
-docs/html
-
+docs/_build
```
Comment (mauritsvanrees) [inline .gitignore:19]: The whole docs dir can be removed from `.gitignore.

File: news/114.documentation [hunk 2]
```diff
@@ -0,0 +1 @@
+Convert README and CHANGES to markdown[ericof]
```
Comment (mauritsvanrees) [inline news/114.documentation:1]: Add a dot and a space (or two):

```suggestion
Convert README and CHANGES to markdown.  [ericof]
```

**Review by mauritsvanrees — APPROVED** [review]
Mostly LGTM.  I added two minor inline comments, but I will approve already.

I think next time we update the configuration using plone/meta, a few changes from `pyproject.toml` and `setup.cfg` will be overwritten.  There are ways around that, editing `.meta.toml`, but maybe not for all.  We can look at that some other time.  Fine for now I would say.

**Pull request merged**